src/framework/vst/internal/synth/vstaudioclient.cpp

```diff
@@ -0,0 +1,153 @@
+#include "vstaudioclient.h"
+
+#include "audio/synthtypes.h"
+
+using namespace mu;
+using namespace mu::vst;
+
+VstAudioClient::~VstAudioClient()
+{
+    IAudioProcessorPtr processor = m_pluginComponent;
+    if (!processor) {
+        return;
+    }
+
+    processor->setProcessing(false);
+    m_pluginComponent->setActive(false);

```

Comment:
A bit strange and confusing code. Better or everywhere to use a class member or variable